mailpoet/assets/js/src/automation/integrations/mailpoet/step-controls/index.tsx

```diff
@@ -33,9 +33,9 @@ export function registerStepControls() {
     (
       controls: StepMoreControlsType,
       step: Step,
-      stepType: StepType,
+      stepType: StepType | undefined,
     ): StepMoreControlsType => {
-      if (stepType.key === 'mailpoet:send-email') {
+      if (stepType?.key === 'mailpoet:send-email') {
```

Comment:
What if we drop `stepType` from this filter and use `step.key` instead?
The thought process is that even an unknown step must have a `key`. And if someone needs a step type, it may be fetchable using a selector, I guess.

Reply:
That's a good idea. Rebased and fixed in da2dd49. Please be aware of https://github.com/mailpoet/mailpoet-premium/pull/663 as da2dd49 alters the signature of the hook.